components/todo/utils/TodoActions.vue

```diff
@@ -32,10 +34,12 @@ export default {
     deleteTodo() {
       if (this.isTodoSearching) return
       this.$store.dispatch('todos/delete', this.todo.id)
+      toast('success', this.$t('deleted', { item: this.$t('todo') }))
     },
     completeTodo() {
       if (this.isTodoSearching) return
       this.$store.dispatch('todos/complete', this.todo.id)
+      toast('success', this.$t('completed', { item: this.$t('todo') }))
```

Comment:
@sai-wtag bhai, as we're not showing toast on edit, the toast also doesn't show when we complete a editing todo. To reproduce,
- Edit a todo
- While editing, without clicking save, click complete, no toast will be shown. 

Proposed solution
- Add toast for edit as well

Reply:
Thanks Bhai, resolved.